juju/juju
# 2.3 skip noop on settings write

## Description of change
Don't issue txn when writing settings would be a no-op

## QA steps
Bootstrap juju, deploy large bundle, verify that there're no empty txns - connect to mongo and issue db.txns.find({o:[]}).count() - it should be 0.

## Bug reference
https://bugs.launchpad.net/juju/+bug/1738209
